pkg/model/fips.go

```diff
@@ -59,6 +72,11 @@ func EnforceGoCompliance(ctx *gotls.Config) {
 		ctx.CipherSuites = fipsGoCiphers
 		ctx.CurvePreferences = []gotls.CurveID{gotls.CurveP256}
 		return
+	case common_features.FIPS_140_3:
+		ctx.MinVersion = gotls.VersionTLS13
+		ctx.MaxVersion = gotls.VersionTLS13
+		ctx.CipherSuites = fips1403GoCiphers
```

Comment:
This field isn't effective with TLS 1.3: https://pkg.go.dev/crypto/tls#Config

> Note that TLS 1.3 ciphersuites are not configurable.

Reply:
ouch. I overlooked that. and according to https://github.com/golang/go/issues/29349, it will stay that way.

that means we can't really implement fips-140-3 compliance policy the way we used to... you can only be fips-compliant by using a go build that is fips-compliant. urgh

Reply:
Yeah I really don't like the Go team's approach here. Envoy has support for BoringSSL compliance policies which helps, but it's boringssl only and obviously doesn't extend to what the control plane does internally 😢